deployments/helm/dra-example-driver/values.yaml

```diff
@@ -12,7 +12,7 @@ allowDefaultNamespace: false
 imagePullSecrets: []
 image:
   repository: registry.example.com/dra-example-driver
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
This doesn't seem relevant to this PR -- or is it somehow needed for immediate allocation to work properly?

Reply:
It is needed for iterative development, helps pulling fresh build instead of reusing old one, why wouldn't you have it by default?
Strictly speaking it is not related to the immediate allocation functionality, I'll cut it out into separate PR.

Reply:
Done.